src/Runner.Sdk/Util/WhichUtil.cs

```diff
@@ -115,11 +115,15 @@ public static string Which(string command, bool require = false, ITraceWriter tr
                 }
             }
 
-            trace?.Info("Not found.");
```

Comment:
Could be the same message with 'Path' / 'PATH'

Originally, I thought the messages for Win and Nix would be quite different